src/routers/handlers/yourCompanies/manageAuthorisedPeopleHandler.ts

```diff
@@ -61,4 +52,40 @@ export class ManageAuthorisedPeopleHandler extends GenericHandler {
             removeUrl: constants.YOUR_COMPANIES_AUTHENTICATION_CODE_REMOVE_URL
         };
     }
+
+    private async handleCancellation (cancellation: Cancellation) {
+        if (cancellation.cancelPerson === constants.YES) {
+            const isUserRemovedFromCompanyAssociations = (await removeUserFromCompanyAssociations(cancellation.userEmail, cancellation.companyNumber)) === constants.USER_REMOVED_FROM_COMPANY_ASSOCIATIONS;
+            if (isUserRemovedFromCompanyAssociations) {
+                this.viewData.cancelledPerson = cancellation.userEmail;
+            }
+        }
+    }
+
+    private async handleRemoval (removal: Removal) {
+        if (removal.removePerson === constants.CONFIRM) {
+            const isUserRemovedFromCompanyAssociations = (await removeUserFromCompanyAssociations(removal.userEmail, removal.companyNumber)) === constants.USER_REMOVED_FROM_COMPANY_ASSOCIATIONS;
+            if (isUserRemovedFromCompanyAssociations) {
+                this.viewData.removedPerson = removal.userName ? removal.userName : removal.userEmail;
```

Comment:
do we have something called userName? the only thing we have is display name. Display name by default is "Not Provided". Should we not check by user_id?